website_snippet_product_category/static/src/js/snippet.options.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 Tecnativa - Alexandre Díaz
+// License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+odoo.define("website_snippet_carousel_product.snippet_options", function(require) {
```

Comment:
If we define here this snippet options related to another module, shouldn't we add it as a dependency?